packages/frontend/src/help/logics/petri-net.mdx

```diff
@@ -0,0 +1,46 @@
+import { A } from "@solidjs/router";
+import { HelpAnalysisById } from "../logic_help_detail"
+
+## Description
+
+**Petri nets** are traditionally a way of expressing discrete-event dynamical systems with concurrency, or networks of resources and processes, describing how various states (called **places**) relate to one another in terms of **transitions**.
```

Comment:
Minor point, but perhaps "are traditionally a way of expressing" -> "were invented to express" since "traditionally" always has a faint connotation of outdatedness to me.